assets/app/styles/_core.less

```diff
@@ -214,6 +214,21 @@ html, body {
       }
     }
   }
+  .empty-project-msg {
```

Comment:
maybe a less specific class name? we might want to reuse this for other empty state boxes?